Windows builds are b0rked

Getting back to Azure Pipeline build scripts to build Mac and Windows, it looks like the Windows builds have been hosed. I'm trying to install a Windows VM and VC++ to diagnose/fix this...

The gist of the issue(s) are:
```
\src\common\dsp\quadfilterunit.cpp(947): error C2065: 'SINUS_SSE2': undeclared identifier [\surge-vst3.vcxproj]
\src\common\vt_dsp\vt_dsp_endian.h(22): error C3861: '__builtin_bswap32': identifier not found (compiling source file src\common\dsp\Wavetable.cpp) [\surge-vst3.vcxproj]
```

--- a/src/common/dsp/QuadFilterUnit.cpp
+++ b/src/common/dsp/QuadFilterUnit.cpp
@@ -726,7 +726,7 @@ __m128 SINUS_SSE1(__m128 in, __m128 drive)
 
    return x;
 }
-
+#endif
 __m128 SINUS_SSE2(__m128 in, __m128 drive)
 {
    const __m128 one = _mm_set1_ps(1.f);
@@ -769,7 +769,6 @@ __m128 SINUS_SSE2(__m128 in, __m128 drive)
 
    return x;
 }
-#endif
 
 #if !_M_X64
 __m128 ASYM_SSE1(__m128 in, __m128 drive)

--- a/src/common/vt_dsp/vt_dsp_endian.h
+++ b/src/common/vt_dsp/vt_dsp_endian.h
@@ -19,7 +19,9 @@ inline float vt_write_float32LE(float f)
 inline int vt_write_int32BE(int t)
 {
 #if (__linux__ || MAC || _M_X64)
-   return __builtin_bswap32(t);
+   // this was `swap_endian`:
+   return ((t << 24) & 0xff000000) | ((t << 8) & 0x00ff0000) | ((t >> 8) & 0x0000ff00) |
+          ((t >> 24) & 0x000000ff);
 #else
    return _byteswap_ulong(t);
 #endif

